From: "Matthew Wilcox (Oracle)" <willy@infradead.org>
Subject: mm: remove unnecessary pagevec includes
Date: Wed, 21 Jun 2023 17:45:57 +0100

These files no longer need pagevec.h, mostly due to function declarations
being moved out of it.

Link: https://lkml.kernel.org/r/20230621164557.3510324-14-willy@infradead.org
Signed-off-by: Matthew Wilcox (Oracle) <willy@infradead.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/fadvise.c        |    1 -
 mm/memory_hotplug.c |    1 -
 mm/migrate.c        |    1 -
 mm/readahead.c      |    1 -
 mm/swap_state.c     |    1 -
 5 files changed, 5 deletions(-)

--- a/mm/fadvise.c~mm-remove-unnecessary-pagevec-includes
+++ a/mm/fadvise.c
@@ -14,7 +14,6 @@
 #include <linux/mm.h>
 #include <linux/pagemap.h>
 #include <linux/backing-dev.h>
-#include <linux/pagevec.h>
 #include <linux/fadvise.h>
 #include <linux/writeback.h>
 #include <linux/syscalls.h>
--- a/mm/memory_hotplug.c~mm-remove-unnecessary-pagevec-includes
+++ a/mm/memory_hotplug.c
@@ -13,7 +13,6 @@
 #include <linux/pagemap.h>
 #include <linux/compiler.h>
 #include <linux/export.h>
-#include <linux/pagevec.h>
 #include <linux/writeback.h>
 #include <linux/slab.h>
 #include <linux/sysctl.h>
--- a/mm/migrate.c~mm-remove-unnecessary-pagevec-includes
+++ a/mm/migrate.c
@@ -21,7 +21,6 @@
 #include <linux/buffer_head.h>
 #include <linux/mm_inline.h>
 #include <linux/nsproxy.h>
-#include <linux/pagevec.h>
 #include <linux/ksm.h>
 #include <linux/rmap.h>
 #include <linux/topology.h>
--- a/mm/readahead.c~mm-remove-unnecessary-pagevec-includes
+++ a/mm/readahead.c
@@ -120,7 +120,6 @@
 #include <linux/export.h>
 #include <linux/backing-dev.h>
 #include <linux/task_io_accounting_ops.h>
-#include <linux/pagevec.h>
 #include <linux/pagemap.h>
 #include <linux/psi.h>
 #include <linux/syscalls.h>
--- a/mm/swap_state.c~mm-remove-unnecessary-pagevec-includes
+++ a/mm/swap_state.c
@@ -16,7 +16,6 @@
 #include <linux/pagemap.h>
 #include <linux/backing-dev.h>
 #include <linux/blkdev.h>
-#include <linux/pagevec.h>
 #include <linux/migrate.h>
 #include <linux/vmalloc.h>
 #include <linux/swap_slots.h>
_
